src/de/arg/mod.rs

```diff
@@ -173,6 +174,7 @@ impl<'a> Visitor<'a> for ArgVisitor<'a> {
                     (value_delimiter, char),
                     (value_name, &str),
                     ref (value_names, Vec<&str>),
+                    // value_parser
```

Comment:
FYI in terms of clap 3.2 support, this is missing ArgAction

Reply:
`value_parser` is implemented on line 231, the specialized part.

Reply:
A value parser describes what type to convert an `OsStr` to.  An `ArgAction` tells clap how to store it in `ArgMatches`.  They fill different roles.

Reply:
Oh, I missed that.
Done in #43.